.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+## Description
+
+
+## Related issues
+
+* none
```

Comment:
Encourage use of "Fixes ..." when PR supposed to close an issue.

Reply:
Should I add a comment for that or a into the template directly

```
* Fixes #issuenumber
```

Reply:
`Fixes #issuenumber` -- like that. Doesn't require a bullet.
And add some comment to remove this line if there is no issue.

Reply:
Updated as discussed with @electrum